SpinCore_pp/ppg/generic.py

```diff
@@ -32,6 +32,7 @@ def generic(
     ret_data=None,
     amplitude=1.0,
     manual_echoes=0,
+    # PR this guy needs to be changed too!
```

Comment:
this

Reply:
see [here](https://jmfrancklab.slack.com/archives/CV553TPRV/p1723756515412379)